packages/exchange/test/api-orders.e2e-spec.ts

```diff
@@ -149,14 +160,49 @@ describe('account ask order send', () => {
     });
 
     it('should be able to withdraw after confirming deposit', async () => {
-        await confirmDeposit();
-
+        const { generationFrom, generationTo, deviceId } = dummyAsset;
```

Comment:
This is good, but there is already a function in `utils.ts` that does that for you:  https://github.com/energywebfoundation/origin/blob/bb309cae2ed8af868568f82b98f1ae7d739d005f/packages/exchange/test/utils.ts#L8 so please use it if possible.

Reply:
Thanks. Its simplifies things a lot